Extensions/TileMap/tilemapruntimeobject.ts

```diff
@@ -12,7 +12,7 @@ namespace gdjs {
     _tilesetJsonFile: string;
     _tilemapAtlasImage: string;
     _displayMode: string;
-    _layerIndex: float;
+    _layerIndex: number;
```

Comment:
Please keep using float, it's more precise than number.

Reply:
the index value is never a float though. It really is an integer

Reply:
Oops my bad! Use `integer` then :)